### PR TITLE
Add AutoDL Models

### PR DESCRIPTION
**Pull request by lixingjian**
I have submitted two AutoDL models and the corresponding codes. These models are generated by Baidu Big Data Lab.

### CODE REVIEW

**Review by qingqing01 — COMMENTED** [review]
Good job. Thanks for your contribution.  Whether we can get the accuracy by eval.py ? @lixingjian

File: PaddleCV/image_classification/models/autodl.py [hunk 1]
```diff
@@ -0,0 +1,563 @@
+#  Copyright (c) 2019 PaddlePaddle Authors. All Rights Reserve.
```
Comment (shippingwang) [inline PaddleCV/image_classification/models/autodl.py]: 2020

File: PaddleCV/image_classification/models/autodl.py [hunk 2]
```diff
@@ -0,0 +1,563 @@
...
+        out = fluid.layers.adaptive_pool2d(s1, (1, 1), "avg")
+        self.logits = fluid.layers.fc(out,
+                                      size=self.class_num,
+                                      param_attr=ParamAttr(
+                                          initializer=Normal(scale=1e-4),
+                                          name='classifier.weight'),
+                                      bias_attr=ParamAttr(
+                                          initializer=Constant(0.),
+                                          name='classifier.bias'))
+        return self.logits#, self.logits_aux
```
Comment (shippingwang) [inline PaddleCV/image_classification/models/autodl.py]: remove comment

File: PaddleCV/image_classification/README.md [hunk 3]
```diff
@@ -602,6 +602,13 @@ python -m paddle.distributed.launch train.py \
 |[ShuffleNetV2_x2_0](https://paddle-imagenet-models-name.bj.bcebos.com/ShuffleNetV2_x2_0_pretrained.tar) | 73.15% | 91.20% | 6.430 | 3.954 |
 |[ShuffleNetV2_swish](https://paddle-imagenet-models-name.bj.bcebos.com/ShuffleNetV2_swish_pretrained.tar) | 70.03% | 89.17% | 6.078 | 4.976 |
 
+### AutoDL Series
```
Comment (shippingwang) [inline PaddleCV/image_classification/README.md:605]: and pls add an update note

File: PaddleCV/image_classification/README.md [hunk 4]
```diff
@@ -602,6 +602,13 @@ python -m paddle.distributed.launch train.py \
...
+|Model | Top-1 | Top-5 | Paddle Fluid inference time(ms) | Paddle TensorRT inference time(ms) |
+|- |:-: |:-: |:-: |:-: |
+|[DARTS_4M](https://paddle-imagenet-models-name.bj.bcebos.com/DARTS_GS_4M_pretrained.tar) | 75.23% | 92.15% |  |  |
+|[DARTS_6M](https://paddle-imagenet-models-name.bj.bcebos.com/DARTS_GS_6M_pretrained.tar) | 76.03% | 92.79% |  |  |
+- AutoDL基于可微结构搜索思路DARTS改进，引入Local Rademacher Complexity控制过拟合，并通过Resource Constraining灵活调整模型大小。
```
Comment (shippingwang) [inline PaddleCV/image_classification/README.md:610]: reference?

**Comment by lixingjian** [comment]
> Good job. Thanks for your contribution. Whether we can get the accuracy by eval.py ? @lixingjian

Yes, we can get the accuracy by "python eval.py --model=AutoDL_4M --pretrained_model=${model_dir}"

File: PaddleCV/image_classification/README.md (hunk 4 above)
Comment (lixingjian) [reply PaddleCV/image_classification/README.md:610]: I have added DARTS as the reference.

**Review by qingqing01 — APPROVED** [review]
